data/card_definitions.json

```diff
@@ -6284,8 +6284,136 @@
+                            "conditions": [{ "condition": "performer_is_specific_id", "required_id": "REPLACED_BY_EFFECT" }],
+                            "effect_type": "power_boost",
+                            "amount": 20
+                        }
+                    }
+                ]
+            },
+            {
+                "skill_id": "birthdaygiftgreen",
+                "cost": 2,
+                "limit": "once_per_game",
+                "effects": [
+                    {
+                        "hide_effect_text": true,
+                        "effect_type": "oshi_activation",
+                        "limit": "once_per_game",
+                        "skill_id": "birthdaygiftgreen",
+                        "pre_effects": [
+                            { "effect_type": "spend_holopower", "amount": 2 },
+                            { "effect_type": "record_used_once_per_game_effect", "effect_id": "birthdaygiftgreen" }
+                        ]
+                    },
+                    {
+                        "effect_type": "choose_cards",
+                        "from": "deck",
+                        "look_at": -1,
+                        "destination": "hand",
+                        "amount_min": 1,
+                        "amount_max": 1,
+                        "requirement": "color_in",
+                        "requirement_colors": ["green"],
+                        "reveal_chosen": true,
+                        "remaining_cards_action": "shuffle"
+                    }
+                ]
+            }
         ]
     },
+    {
+        "card_id": "hPR-001",
+        "card_type": "holomem_spot",
+        "bloom_blocked": true,
+        "card_names": ["sakura_miko"],
+        "rarity": "p",
+        "colors": ["none"],
+        "hp": 50,
+        "baton_cost": 1,
+        "tags": [
+            "#JP",
+            "#Gen0",
+            "#Song"
+        ],
+        "arts": [
+            {
+                "art_id": "flowerrhapsody",
+                "costs": [{ "color": "any", "amount": 1 }],
+                "power": 10
+            }
+        ],
+        "collab_effects": [
+            {
+                "effect_type": "choice",
+                "choice": [
+                    {
+                        "effect_type": "roll_die",
+                        "source": "holomem_ability",
+                        "die_effects": [
+                            {
+                                "english_values": "odd",
+                                "activate_on_values": [1, 3, 5],
+                                "effects": [
+                                    {
+                                        "effect_type": "choose_cards",
+                                        "from": "cheer_deck",
+                                        "look_at": -1,
+                                        "destination": "holomem",
+                                        "to_limitation": "backstage",
```

Comment:
can you double check that the right card text shows up for this on the card?
I think to_limitation may be missing in strings.gd from the string builder portion of choose_cards

Reply:
I've added the `to_limitation` strings for `choose_cards` and applied the other cases that were being handled the same way like `tag_in` and such. 

Along the way, I noticed that localization were not applied to tags since those are newer additions so I added those in as well.

<img width="250" alt="screenshot 12" src="https://github.com/user-attachments/assets/e21a78a9-ceab-4c83-9e97-fbf420bc2232">
<img width="250" alt="screenshot 13" src="https://github.com/user-attachments/assets/4eac9bb8-3c6d-490b-bd29-7568fdcd8c82">

Reply:
Did we actually want to translate the tags?
It's kind of weird to translate hashtags.
I don't have super strong opinions either way though.

Reply:
I think it's better since the kr proxies translated theirs as well. And the translations are available anyway, so it was just a small change.